backend/helpers/emailService.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require('nodemailer');
+
+const { EMAIL_USER, EMAIL_PASS, EMAIL_HOST, SENDER_EMAIL } = require('../config/index');
+const { getMailTemplate } = require('../utility/emailTemplates');
+
+const transporter = nodemailer.createTransport({
+  type: 'SMTP',
+  host: EMAIL_HOST,
+  secure: true,
+  debug: true,
+  port: 465,
+  auth: {
+    user: EMAIL_USER,
+    pass: EMAIL_PASS,
+  },
+});
+
+module.exports.sendEmail = async (email, type) => {
+  const { subject, text } = getMailTemplate(type);
+  const mailOptions = {
+    from: SENDER_EMAIL,
+    to: email,
+    subject,
+    text,
+  };
+  try {
+    await transporter.sendMail(mailOptions);
+  } catch (err) {
+    console.log(err);
```

Comment:
Please follow our error handling mechanism

Reply:
In the error constants do I add another one called EMAIL ERROR, or UNEXPECTED ERROR. As There can be multiple errors that happen while sending an email.

Reply:
Check the updated link. Revert if it doesn't work